### DISCUSSION TITLE
Multiple GitHub authentication connector

### DISCUSSION THREAD

**Discussion by colussim** [gravitational/teleport#22715] [Q&A] [answered]
Hi,
I would like to know if it is possible to configure several GitHub connectors to authenticate.
I have a pool of different customer who needs to connect to a Kubernetes cluster in their namespaces, the authentication should be done by their GitHub account. Is it possible?

Thanks
Emmanuel

**Comment by webvictim** [comment, marked as answer]
Yes, you can configure multiple Github connectors in Teleport. You'd also need to register separate Github applications, each with permissions under a different organisation. You can then use the `metadata.name` field inside the connector to separate the connectors - i.e:

```yaml
metadata:
  name: github-organisation1
```

```yaml
metadata:
  name: github-organisation2
```

**Reply by colussim** [reply]
Many Thanks
Emmanuel
